learn/best_practices/index.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: book
+title: Fortran Best Practices
+permalink: /learn/best_practices
+author: Ondřej Čertík, John Pask, Jed Brown, Matthew Emmett, Juan Luis Cano Rodríguez, Neil Carlson, Andrea Vigliotti, Pierre Haessig
+---
+
+This mini-book collects a modern canonical way of doing things in Fortran. It
+is meant to be short, and it is assumed that you already know how to
+program in other languages (like Python, C/C++, ...) and also know
+Fortran syntax a bit. Some things in Fortran are obsolete, so this guide
+only shows the "one correct/canonical modern way" how to do things.
+
+Summary of the language:
+<http://www.cs.umbc.edu/~squire/fortranclass/summary.shtml>
```

Comment:
Link should be marked as external

Reply:
Removed in https://github.com/fortran-lang/fortran-lang.org/pull/246/commits/5cf80cc2ecb95001247d47e89b53910068aafbeb